.github/issue_template.md

```diff
@@ -0,0 +1,15 @@
+# Title/Heading
+
+Please include a small description or summary of the issue here.
+
+## Bug Reports
+Please replace this line with a brief summary of your issue AND if reporting a build issue include the link.
+
+[ ] Didn't run?
+[ ] Permission Issues?
+[ ] Broken Links
+[ ] Other Issues? (Add Here)
+
+## Want to help solve this Issue?
```

Comment:
Replace this section with an additional resources section where any links, references are can be put by the user

Reply:
On it!